website/www/site/content/en/get-started/downloads.md

```diff
@@ -96,10 +96,17 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.58.0 (2024-07-22)
+Official [source code download](https://downloads.apache.org/beam/2.57.0/apache-beam-2.58.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.57.0/apache-beam-2.58.0-source-release.zip.sha512).
```

Comment:
use 2.58.0 (multiple places in URLs need a tweak)

Reply:
fixed